eng/Versions.props

```diff
@@ -35,6 +35,8 @@
     <MicrosoftNETCorePlatformsVersion>2.1.0</MicrosoftNETCorePlatformsVersion>
     <MicrosoftNetCompilersToolsetVersion>4.1.0-1.21507.14</MicrosoftNetCompilersToolsetVersion>
     <MicrosoftNetTestSdkVersion>15.7.2</MicrosoftNetTestSdkVersion>
+    <MicrosoftVisualStudioDiagnosticsUtilitiesVersion>17.5.0-preview-3-33219-258</MicrosoftVisualStudioDiagnosticsUtilitiesVersion>
+    <MicrosoftVisualStudioEnterpriseAspNetHelper>17.5.0-preview-3-33219-258</MicrosoftVisualStudioEnterpriseAspNetHelper>
```

Comment:
Shall the versions be kept in sync? If so I'd recommend the second variable to use the first one.

Reply:
Done, thanks !